### PR TITLE
fix: Fixed nullable annotations for WeakEventManager.

### PR DESCRIPTION
**Pull request by HavenDV**
### Description of Change
By the logic of working with all events, this seems to have been missed when adding the #nullable enable annotation.
This PR fixes nullable warnings when working with the WeakEventManager.HandleEvent method.
I'm not sure if there is anything in the documentation that needs to be changed.

### Issues Fixed

Fixes #

### CODE REVIEW

**Review by jfversluis — CHANGES_REQUESTED** [review]
Thanks for this @HavenDV!

You will also need to update these files, although I can understand if it might not be immediately clear for you how to do it 😅

>D:\a\_work\1\s\src\Core\src\WeakEventManager.cs(40,15): error RS0016: Symbol 'HandleEvent' is not part of the declared public API [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=netstandard2.0]
D:\a\_work\1\s\src\Core\src\PublicAPI\netstandard2.0\PublicAPI.Shipped.txt(1712,1): error RS0017: Symbol 'Microsoft.Maui.WeakEventManager.HandleEvent(object! sender, object! args, string! eventName) -> void' is part of the declared API, but is either not public or could not be found [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=netstandard2.0]
D:\a\_work\1\s\src\Core\src\WeakEventManager.cs(40,15): error RS0016: Symbol 'HandleEvent' is not part of the declared public API [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=netstandard2.1]
D:\a\_work\1\s\src\Core\src\PublicAPI\netstandard\PublicAPI.Shipped.txt(1714,1): error RS0017: Symbol 'Microsoft.Maui.WeakEventManager.HandleEvent(object! sender, object! args, string! eventName) -> void' is part of the declared API, but is either not public or could not be found [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=netstandard2.1]
D:\a\_work\1\s\src\Core\src\WeakEventManager.cs(40,15): error RS0016: Symbol 'HandleEvent' is not part of the declared public API [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0]
D:\a\_work\1\s\src\Core\src\PublicAPI\net\PublicAPI.Shipped.txt(1714,1): error RS0017: Symbol 'Microsoft.Maui.WeakEventManager.HandleEvent(object! sender, object! args, string! eventName) -> void' is part of the declared API, but is either not public or could not be found [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0]
D:\a\_work\1\s\src\Core\src\WeakEventManager.cs(40,15): error RS0016: Symbol 'HandleEvent' is not part of the declared public API [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0-ios]
D:\a\_work\1\s\src\Core\src\PublicAPI\net-ios\PublicAPI.Shipped.txt(1959,1): error RS0017: Symbol 'Microsoft.Maui.WeakEventManager.HandleEvent(object! sender, object! args, string! eventName) -> void' is part of the declared API, but is either not public or could not be found [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0-ios]
D:\a\_work\1\s\src\Core\src\WeakEventManager.cs(40,15): error RS0016: Symbol 'HandleEvent' is not part of the declared public API [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0-maccatalyst]
D:\a\_work\1\s\src\Core\src\PublicAPI\net-maccatalyst\PublicAPI.Shipped.txt(1958,1): error RS0017: Symbol 'Microsoft.Maui.WeakEventManager.HandleEvent(object! sender, object! args, string! eventName) -> void' is part of the declared API, but is either not public or could not be found [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0-maccatalyst]
D:\a\_work\1\s\src\Core\src\WeakEventManager.cs(40,15): error RS0016: Symbol 'HandleEvent' is not part of the declared public API [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0-tizen]
D:\a\_work\1\s\src\Core\src\PublicAPI\net-tizen\PublicAPI.Shipped.txt(1938,1): error RS0017: Symbol 'Microsoft.Maui.WeakEventManager.HandleEvent(object! sender, object! args, string! eventName) -> void' is part of the declared API, but is either not public or could not be found [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0-tizen]
D:\a\_work\1\s\src\Core\src\WeakEventManager.cs(40,15): error RS0016: Symbol 'HandleEvent' is not part of the declared public API [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0-windows10.0.20348]
D:\a\_work\1\s\src\Core\src\PublicAPI\net-windows\PublicAPI.Shipped.txt(1919,1): error RS0017: Symbol 'Microsoft.Maui.WeakEventManager.HandleEvent(object! sender, object! args, string! eventName) -> void' is part of the declared API, but is either not public or could not be found [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0-windows10.0.20348]
D:\a\_work\1\s\src\Core\src\WeakEventManager.cs(40,15): error RS0016: Symbol 'HandleEvent' is not part of the declared public API [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0-windows10.0.19041]
D:\a\_work\1\s\src\Core\src\PublicAPI\net-windows\PublicAPI.Shipped.txt(1919,1): error RS0017: Symbol 'Microsoft.Maui.WeakEventManager.HandleEvent(object! sender, object! args, string! eventName) -> void' is part of the declared API, but is either not public or could not be found [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0-windows10.0.19041]
D:\a\_work\1\s\src\Core\src\WeakEventManager.cs(40,15): error RS0016: Symbol 'HandleEvent' is not part of the declared public API [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0-android]
D:\a\_work\1\s\src\Core\src\PublicAPI\net-android\PublicAPI.Shipped.txt(2044,1): error RS0017: Symbol 'Microsoft.Maui.WeakEventManager.HandleEvent(object! sender, object! args, string! eventName) -> void' is part of the declared API, but is either not public or could not be found [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net7.0-android]

Basically you want to add a line like this to each of the unshipped txt files

> \*REMOVED\*Microsoft.Maui.WeakEventManager.HandleEvent(object! sender, object! args, string! eventName) -> void

Because the non-nullable version of this API has been removed.

And then you need to also add one like this to the unshipped txt files

> Microsoft.Maui.WeakEventManager.HandleEvent(object? sender, object? args, string! eventName) -> void

Because you're adding the nullable one. Let me know if you can't figure it out :)

**Review by mattleibow — CHANGES_REQUESTED** [review]
Thanks for the PR. The Shipped files are generated by a tool/script, so your changes may be lost. You have a \*REMOVE\* in the unshipped which is correct, but you also need to add the NEW api into the unshipped instead of changing the shipped file.

**Review by mattleibow — APPROVED** [review]

**Review by jfversluis — APPROVED** [review]

**Comment by jfversluis** [comment]
Thank you so much for your very first contribution @HavenDV looking forward to the next ones! 

Here is virtual cupcakes 🧁🧁🧁🧁 to celebrate!
